requirements-dev.txt

```diff
@@ -22,6 +22,8 @@ pytest-xdist>=3
 responses>=0.23.1
 selenium>=4.16.0
 sentry-cli>=2.16.0
+html2text==2024.2.26
+djlint==1.36.4
```

Comment:
@asottile-sentry What's the right way to add dev deps? These are not in https://pypi.devinfra.sentry.io/simple